### PR TITLE
EASY-1291

### PR DESCRIPTION
**Pull request by rvanheest**
Fixes EASY-1291

#### When applied it will
* update dependent role version
* fix the yml file
* put a better reasonable default in the properties
* fixed the logback file
* fixed the rpm scripts

@DANS-KNAW/easy for review

### CODE REVIEW

File: src/main/assembly/dist/cfg/application.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@ deposits.permissions=rwxrwx---
 tempdir=/var/opt/dans.knaw.nl/tmp/easy-sword2
 # Require at least some 2 G  spare disk space
 tempdir.margin-available-diskspace-mb=2000
-base-url=http://yourdomain-here.org/
+base-url=http://test.dans.knaw.nl/
```
Comment (janvanmansum) [inline src/main/assembly/dist/cfg/application.properties]: I disagree. When creating "reasonable defaults" picture the scenario of someone installing the rpm via a `yum install`, but you don't know anything else about them or their environment. The base-url of their server is therefore something that you really cannot predict.

File: src/main/ansible/vagrant.yml [hunk 2]
```diff
@@ -23,29 +23,31 @@
...
-      yum:
-        name: dans.knaw.nl-easy-sword2
-        state: latest
+  - name: Create directories
+    file:
+      path: "{{ item }}"
+      recurse: yes
+      state: directory
+      mode: "0777"
+    with_items:
```
Comment (janvanmansum) [inline src/main/ansible/vagrant.yml]: I am in favor of keeping the creation of these default directories in the RPM post install.

**Review by janvanmansum — CHANGES_REQUESTED** [review]
I want to keep the creation of default dirs in the RPM. The rest seems OK.

File: src/main/assembly/dist/cfg/application.properties (hunk 1 above)
Comment (rvanheest) [reply src/main/assembly/dist/cfg/application.properties]: mmm, maybe you're right. I was thinking the 'reasonable default' should cater towards the local vm. Also I don't expect anyone else to use this software, other than us...

File: src/main/ansible/vagrant.yml (hunk 2 above)
Comment (rvanheest) [reply src/main/ansible/vagrant.yml]: I'm not. I think both configuration of these directories and creating them should be left to the deployer, e.g. in ansible/vagrant. Imagine the deployer wishes to put these directories in a different place, now he first has to remove the old directory (which he might not have the rights to?), then create the new directory and set the configurations correctly. I think we should keep the rpm install for what it currently does: stopping the service, installing the software and restarting the service.

**Review by janvanmansum — APPROVED** [review]

**Pull request merged**